src/universal/components/EditingStatus/EditingStatus.js

```diff
@@ -61,4 +65,13 @@ const styleThunk = (custom, {isEditing}) => ({
   }
 });
 
-export default withStyles(styleThunk)(EditingStatus);
+export default createFragmentContainer(
```

Comment:
So hear me out, how I think of this is there is an `EditingStatus` component that is wrapped in a container, and that container is nothing more than the thing the HOC provides. This is the relay-approved way of doing things because the fragment name must match the component name that it's tied to, so if we tied it to EditingStatusContainer, we'd be breaking that convention.